xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -19,8 +19,18 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides a high-performance API for forward-only, non-cached writing of UTF-8 encoded JSON text. It writes the text sequentially with no caching and by default adheres to the <see href="https://tools.ietf.org/html/rfc8259">JSON RFC</see>, with the exception of writing comments.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+When the user attempts to write invalid JSON and validation is enabled, it throws a <xref:System.InvalidOperationException> with a context specific error message.
+Since this type is a ref struct, it does not directly support async. However, it does provide support for reentrancy to write partial data, and continue writing in chunks.
```

Comment:
```suggestion
Since this type is a ref struct, it does not directly support async. However, it does provide support for reentrancy to write partial data and to continue writing in chunks.
```